RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -139,23 +139,6 @@ - (void)syncPurchases {
     }];
 }
 
-- (void)addAttributionData:(NSString *)dataJSON network:(int)network networkUserId:(NSString * _Nullable)networkUserId {
```

Comment:
this should have been deleted in 4.0.0, my bad! It was unused, though. And it's obsoleted in C#